README.md

```diff
@@ -41,7 +42,7 @@ Config example using a querystring:
 module.exports = {
   module: {
     loaders: [
-      { test: /\.ejs$/, loader: 'ejs-loader?variable=data' },
+      { test: /\.ejs$/, loader: '@groveco/ejs-loader?variable=data' },
```

Comment:
This entire section should be removed, as passing options via querystring was deprecated in Webpack 3.

Reply:
@racoleman it's deprecated but technically someone could use this package with older versions of webpack. My guess is that if people are still using `ejs` templates, they could also be using older versions of webpack

Reply:
If you wish to maintain documentation for older versions, I'd suggest putting this under a "Webpack 2" header and make sure the first example is written to Webpack 4.